libs/model-ad/not-found/src/lib/not-found.component.ts

```diff
@@ -0,0 +1,36 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+import { RouterModule } from '@angular/router';
+import { MatCardModule } from '@angular/material/card';
+// import { ConfigService } from '@sagebionetworks/openchallenges/config';
+import { FooterComponent } from '@sagebionetworks/model-ad/ui';
+
+@Component({
+  selector: 'openchallenges-not-found',
```

Comment:
This may need to be updated if we decide to update the prefixes in `libs/model-ad/not-found/.eslintrc.json`

Reply:
Fixed